src/services/agent-service.ts

```diff
@@ -92,7 +92,9 @@ export class AgentService {
     const configuration = new ConfigurationService().configuration
     const snapshotOptions: SnapshotOptions = {
       widths: request.body.widths || configuration.snapshot.widths,
-      enableJavaScript: request.body.enableJavaScript,
+      enableJavaScript: request.body.enableJavaScript != null
+        ? request.body.enableJavaScript
+        : configuration.snapshot['enable-javascript'],
```

Comment:
The [nullish operator](https://github.com/microsoft/TypeScript/issues/26578) would've been perfect here:

```js
request.body.enableJavaScript ?? configuration.snapshot['enable-javascript']
```